### PR TITLE
Bump k8s.io/kubernetes to 1.19.7 and add back InstancesV2

### PR DESCRIPTION
**Pull request by nicolehanjing**
**What this PR does / why we need it**:
Bump k8s.io/kubernetes to 1.19.7 and add back InstancesV2

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Bump k8s.io/kubernetes to 1.19.7 and add back InstancesV2
```

### CODE REVIEW

File: pkg/nsxt/connector_manager.go [hunk 1]
```diff
@@ -194,6 +194,9 @@ func (cm *ConnectorManager) GetConnector() client.Connector {
 
 // AddSecretListener adds secret informer add, update, delete callbacks
 func (cm *ConnectorManager) AddSecretListener(secretInformer v1.SecretInformer) error {
+	if cm.config == nil {
```
Comment (nicolehanjing) [inline pkg/nsxt/connector_manager.go:197]: @andrewsykim received nil pointer errors if the config is not set, thinking we should add this here, or should I open another PR for this fix?

**Comment by dvonthenen** [comment]
When this PR gets merged, please also review the PR for including YAML for 1.19 as well.
https://github.com/kubernetes/cloud-provider-vsphere/pull/415

File: cluster/images/controller-manager/Dockerfile [hunk 2]
```diff
@@ -14,7 +14,7 @@
 ##                               BUILD ARGS                                   ##
 ################################################################################
 # This build arg allows the specification of a custom Golang image.
-ARG GOLANG_IMAGE=golang:1.13.14
+ARG GOLANG_IMAGE=golang:1.15.6
```
Comment (andrewsykim) [inline cluster/images/controller-manager/Dockerfile:17]: Is this the GO version used in v1.19.7?

File: pkg/nsxt/connector_manager.go [hunk 3]
```diff
@@ -194,6 +194,9 @@ func (cm *ConnectorManager) GetConnector() client.Connector {
...
+		return errors.New("Config is not available for NSXT connector manager")
```
Comment (andrewsykim) [inline pkg/nsxt/connector_manager.go:198]: this diff can be removed now

File: cluster/images/controller-manager/Dockerfile (hunk 2 above)
Comment (nicolehanjing) [reply cluster/images/controller-manager/Dockerfile:17]: @andrewsykim 
I took a look at https://github.com/kubernetes/kubernetes/blob/v1.19.7/go.mod#L9 and it's using `go 1.15` so I just search the latest Golang image version 1.15.6 and use it here, is that the right way to update it?

Comment (andrewsykim) [reply cluster/images/controller-manager/Dockerfile:17]: yeah that seems good enough, just want to make sure we are on the same minor at least. Patch version is _probably_ not as important.

**Review by andrewsykim — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**